lib/terminus-bootstrap.service.ts

```diff
@@ -102,7 +102,7 @@ export class TerminusBootstrapService implements OnApplicationBootstrap {
     return this.options.endpoints.reduce((healthChecks, endpoint) => {
       const url = this.validateEndpointUrl(endpoint);
       healthChecks[url] = async () =>
-        this.healthCheckExecutor.executeDeprecated(endpoint.healthIndicators);
+        this.healthCheckExecutor.execute(endpoint.healthIndicators);
```

Comment:
`getHealthChecksMap` will get completley removed in v8. Since the e2e tests are still dependent on this function, we need to revert that for now until the tests have been refactored

```suggestion
        this.healthCheckExecutor. executeDeprecated(endpoint.healthIndicators);
```

Reply:
Done!